streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -225,6 +225,11 @@ void initializeStateStores() {
         }
     }
 
+    @Override
+    public void initializeTopology() {
```

Comment:
Should we add this? It's an abstract class. Maybe it's better for force an implementation in concrete classes?

Reply:
Ack